variables.tf

```diff
@@ -682,6 +682,7 @@ variable "cos" {
 # Service Instance Variables
 ##############################################################################
 
+# tflint-ignore: terraform_unused_declarations
```

Comment:
why are we ignoring this? If it is not used, it should not be here, but why is it not used?

Reply:
it is not used inside patterns, instead the value is take from override our default is set. Will remove it